keyboards/kbdfans/kbd67/rev2/keymaps/default/config.h

```diff
@@ -0,0 +1,19 @@
+/* Copyright 2018 MechMerlin
```

Comment:
I don't remember writing these files =P

Reply:
@mechmerlin These are all copied from the `rev1` files, so you actually *did* write them.

Reply:
@mechmerlin @noroadsleft Correct, which is why I left the original author.